lib/src/client_task_impl.dart

```diff
@@ -17,8 +17,6 @@ class ClientTaskImpl implements TaskInfo {
 
   @override
   Future<Task> getStatus() async {
```

Comment:
Just a little question here why is this one inside the `getStatus` method because I understand that I can call `client.getTask` but this construction is not clear for me.

Reply:
Since we have access to the `client` from the `ClientTaskImpl` and the client has the `getTask` method, I just opted to call this `getTask` directly in order to not duplicate the code (as I did before). I don’t know if this answers your question @alallema (if not please ask again 😄).